lib/postgres.go

```diff
@@ -2916,3 +2944,50 @@ func (postgres *Postgres) GetNotifications(publicKey string) ([]*PGNotification,
 
 	return notifications, nil
 }
+
+//
+// Postgres Test tooling
+//
+
+// Drop all tables in the postgres Db.
+func (postgres *Postgres) resetDatabase() error {
+	res, err := postgres.db.Exec("DROP SCHEMA public CASCADE; CREATE SCHEMA public;")
+	if err != nil {
+		return errors.Wrapf(err, "DeletePostgresDatabase: Problem executing the query")
+	}
+	glog.V(1).Infof(CLog(Blue, fmt.Sprintf("DeletePostgresDatabase: Got res %v", res)))
+	return nil
+}
+
+// Kill the postgres Docker container and delete the postgres directory. This is used for running pg tests.
+func (postgres *Postgres) forceKillContainer() error {
+	if postgres.containerId == "" {
+		return fmt.Errorf("forceKillContainer: container Id is empty. This function only works for dockerized postgres")
+	}
+
+	err := postgres.db.Close()
+	if err != nil {
+		return errors.Wrapf(err, "forceKillContainer: Problem closing postgres db")
+	}
+
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		return errors.Wrapf(err, "forceKillContainer: Problem creating new docker client")
+	}
+
+	removeOptions := types.ContainerRemoveOptions{
+		Force: true,
+	}
+	err = cli.ContainerRemove(context.Background(), postgres.containerId, removeOptions)
+	if err != nil {
+		return errors.Wrapf(err, "forceKillContainer: Problem removing postgres container")
+	}
+
+	if postgres.directory == "" {
```

Comment:
why even symlink the postgres data at all. if it's just for testing, may as well not mount the volume IMO.

Reply:
agreed^